app/assets/about.md

```diff
@@ -127,3 +127,16 @@ stateCases = tryCatch(
 ```
   
   
+
+##### Forecast evaluation
```

Comment:
suggestion: More descriptive section heading would be nice. The difference in format is small, but maybe?

```suggestion
##### Forecasts with actuals
```